### PR TITLE
Chore: expressions with type arguments can be replaced a with <> (JDK 9+)

### PR DESCRIPTION
**Pull request by dbmalkovsky**
Ran `mvn clean test` successfully.

### CODE REVIEW

File: modules/flowable-cmmn-engine/src/main/java/org/flowable/cmmn/engine/test/impl/CmmnTestRunner.java [hunk 1]
```diff
@@ -207,16 +207,16 @@ protected void assertDatabaseEmpty(FrameworkMethod method) {
...
             true,
-            new Command<Void>() {
-
-                @Override
-                public Void execute(CommandContext commandContext) {
-                    SchemaManager schemaManager = CommandContextUtil.getCmmnEngineConfiguration(commandContext).getSchemaManager();
-                    schemaManager.schemaDrop();
-                    schemaManager.schemaCreate();
-                    return null;
+                new Command<>() {
```
Comment (filiphr) [inline modules/flowable-cmmn-engine/src/main/java/org/flowable/cmmn/engine/test/impl/CmmnTestRunner.java:210]: This looks like the formatting was changed as well. Is that mandatory?

File: modules/flowable-engine-common/src/main/java/org/flowable/common/engine/impl/persistence/deploy/DefaultDeploymentCache.java [hunk 2]
```diff
@@ -41,21 +41,23 @@ public DefaultDeploymentCache() {
...
-                boolean removeEldest = size() > limit;
-                if (removeEldest && LOGGER.isTraceEnabled()) {
-                    LOGGER.trace("Cache limit is reached, {} will be evicted", eldest.getKey());
-                }
-                return removeEldest;
-            }
-
-        });
+        this.cache = Collections.synchronizedMap(
+                new LinkedHashMap<>(limit + 1, 0.75f, true) { // +1 is needed, because the entry is inserted first, before it is removed
```
Comment (filiphr) [inline modules/flowable-engine-common/src/main/java/org/flowable/common/engine/impl/persistence/deploy/DefaultDeploymentCache.java:45]: Do we need to change the formatting here? Can we perhaps keep only the change only to the `<>`?

**Review by filiphr — COMMENTED** [review]
Thanks @dbmalkovsky. Most of it looks good to me. There are some changes in formatting in few places and I've left a comment for those. Can you please have a look?

btw, this is not a JDK 7+ feature. This is the enhanced diamond operator for JDK  9+, where you do not need to define the parameter in abstract inner classes.

File: modules/flowable-cmmn-engine/src/main/java/org/flowable/cmmn/engine/test/impl/CmmnTestRunner.java (hunk 1 above)
Comment (dbmalkovsky) [reply modules/flowable-cmmn-engine/src/main/java/org/flowable/cmmn/engine/test/impl/CmmnTestRunner.java]: reverted to original foirmatting

File: modules/flowable-engine-common/src/main/java/org/flowable/common/engine/impl/persistence/deploy/DefaultDeploymentCache.java (hunk 2 above)
Comment (dbmalkovsky) [reply modules/flowable-engine-common/src/main/java/org/flowable/common/engine/impl/persistence/deploy/DefaultDeploymentCache.java]: reverted to original formatting

**Comment by filiphr** [comment]
Thanks @dbmalkovsky

**Pull request merged**